app/latigo/log.py

```diff
@@ -56,3 +58,7 @@ def setup_logging(filename, log_level=logging.INFO):
     logger = logging.getLogger(log_filename)
     logger.info(f"Log started for {log_filename}")
     return logger
+
+def add_azure_logging(logger, enable_azure_logging=False, azure_monitor_instrumentation_key=None):
+    if enable_azure_logging:
+            logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=' + azure_monitor_instrumentation_key))
```

Comment:
Also, for debugging might be useful to have following:
```suggestion
            logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=' + azure_monitor_instrumentation_key))
            logger.info("AzureLogHandler was enabled.")
```